File: PRAKTIKUM_4/soal5.py
N = int(input("masukan N : "))
k = int(input("masukan kelipatan : "))

hasil=0
hasilakhir=0
for i in range(1,N+1,+1):
    hasil+=i*k
    hasilakhir+=hasil
    print(f"({i}*{k})",end="")
    for j in range(i-1,0,-1):
        print(f"({j}*{k})",end="")
    print(f" = {hasil}")
print(f" = {hasilakhir}")

File: PRAKTIKUM_4/soal5.c
#include <stdio.h>

int main() {
    int N, kelipatan;
    
    printf("Masukkan N: ");
    scanf("%d", &N);
    
    printf("Masukkan kelipatan: ");
    scanf("%d", &kelipatan);

    int hasil = 0,hasilakhir=0;
    for (int i = 1; i <= N; i++) {
       
        hasil += i * kelipatan;
        hasilakhir+= hasil;
        printf("(%d*%d)", i, kelipatan);
        for(int j=i-1;j>0;j--){
            printf("+(%d*%d)",j,kelipatan);
            
        }
    printf(" = %d\n", hasil);

 
    }printf("%d",hasilakhir);

    

}
